roles/install/tasks/3_storage_config.yml

```diff
@@ -1,62 +1,62 @@
-- name: Storage Config | Ensure lvm2 package is installed.
-  package:
-    name: lvm2
-    state: present
-
-- name: Storage Config | Check if drives are already mounted
-  command: '/bin/mountpoint -q {{ item.mount_point }}'
-  with_items: '{{ satellite_req_dirs }}'
-  register: ebs_checked
-  changed_when: "ebs_checked.rc != 0"
+#- name: Read device information (always use unit when probing)
+#  parted: device=/dev/"{{ item.device }}"
+#  register: sdb_info
+#  with_items:
+#    - "{{ ansible_mounts }}"
 
-#yamllint disable-line rule:line-length
-# https://access.redhat.com/documentation/en-us/red_hat_satellite/6.4/pdf/installing_satellite_server_from_a_connected_network/Red_Hat_Satellite-6.4-Installing_Satellite_Server_from_a_Connected_Network-en-US.pdf
-# See Section 2.1.3 - Table 1.1
+- name: Getting unused disks
+  set_fact:
+    device_name: "{{ item.key }}"
```

Comment:
You're going to get one result here - the last entry, not the whole list of them. Is that what you really want? If you want to run the subsequent block on all elements that match your conditionals, you need to move the block into a file and have this be an include_tasks instead of a set_fact. Then you'll run the subsequent commands for every mount point.